ivy/functional/frontends/torch/loss_functions.py

```diff
@@ -16,3 +53,29 @@ def cross_entropy(
 
 
 cross_entropy.unsupported_dtypes = ("float16",)
+
+
+def binary_cross_entropy(
+    input, 
+    target, 
+    weight=None, 
+    size_average=None, 
+    reduce=None, 
+    reduction='mean'
+):
+    reduction = _get_reduction(reduction, size_average, reduce)
+    result = ivy.binary_cross_entropy(target, input, epsilon=0.0)
+    
+    if weight is not None:
+        result = ivy.multiply(weight, result)
+    result = reduction(result)
+    return result
+
+
+binary_cross_entropy.unsupported_dtypes = (
```

Comment:
are you sure it does not support so many dtypes? did you try to delete one of them and test it?

Reply:
Pytorch's bce does not support integer datatype targets, but ivy's bce does. 
I misunderstood that and wrote `unsupported_dtypes` incorrectly. 
I will modify and retest `unsupported_dtypes` under the assumption that the code of the torch that works normally is changed to ivy. 
The integer datatype will be deleted.